src/commands/help.ts

```diff
@@ -21,7 +20,7 @@ export default {
                 .setFooter(`You can send "${prefix}help [command name]" to get info on a specific command!`);
 
             client.commands.forEach((command) => {
-                if (command.level === 'user') {
+                if (!command.adminOnly && !command.ownerOnly) {
```

Comment:
Shouldn't this be `||` instead of `&&`?

```suggestion
                if (!command.adminOnly || !command.ownerOnly) {
```

Reply:
No | | makes the bot print the entire list of commands because one of the attributes is true and the other is false
&& prints just the non admin/owner only commands